cookbooks/aws-parallelcluster-install/recipes/base.rb

```diff
@@ -99,17 +99,7 @@
   mode '0644'
 end
 
-# Install NFS packages
-if platform?('ubuntu') && node['platform_version'].to_f >= 16.04
-  # FIXME: https://github.com/sous-chefs/nfs/issues/93
-  include_recipe "nfs::server"
-end
-include_recipe "nfs::server4"
-
-# Disable NFS server service start at boot
-service node['nfs']['service']['server'] do
-  action :disable
-end
+nfs 'install NFS daemon' unless virtualized?
```

Comment:
Is `unless virtualized?` still needed?

Reply:
Yes because the `include_recipe "nfs::server4"` tries to enable some services.
Anyway thanks to the comment I decided to move it into the specific actions, because in theory some actions can be executed on docker and some cannot.